src/plugins/analysis/file_system_metadata/test/test_plugin_file_system_metadata.py

```diff
@@ -1,23 +1,28 @@
-# pylint: disable=no-self-use,protected-access
-import os
+# pylint: disable=no-self-use,protected-access,wrong-import-order
 from base64 import b64encode
+from pathlib import Path
+from typing import Optional
 
-from common_helper_files.fail_safe_file_operations import get_dir_of_file
+from flaky import flaky
 
 from test.common_helper import TEST_FW, TEST_FW_2, DatabaseMock, create_test_file_object
+from test.mock import mock_patch
 from test.unit.analysis.analysis_plugin_test_class import AnalysisPluginTest
+
 from ..code import file_system_metadata as plugin
 from ..code.file_system_metadata import FsKeys
 
-TEST_DATA_DIR = os.path.join(get_dir_of_file(__file__), 'data')
+PLUGIN_NAME = 'file_system_metadata'
```

Comment:
Can't you take this from the `NAME` attribute of the plugin?

Reply:
In general: yes. But it is used by the base class `AnalysisPluginTest` (which means we can't really get rid of it except if we change all plugin tests)